testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -186,95 +186,99 @@ def correct_values_backbone_group(self):
         return [[0, 1, 0, 0, 0],
                 [49, 50, 4.6997, 1.9154, 2.7139]]
 
-    def test_rmsd(self, universe, correct_values):
+    def test_rmsd(self, universe, correct_values, client_RMSD):
```

Comment:
When the `client_RMSD` fixture is used for the first time, it's a bit mysterious where it comes from and what it does because it's not defined in the same file.

Add a comment explaining where `client_RMSD` is defined and what it does.